### ISSUE TITLE
Old version for Windows XP

### ISSUE THREAD

**Issue by Dimsok** [FRex/csoinfo#1]
Is there any old version, which i can run on Windows XP? If is, could someone share it?

**Comment by FRex** [comment]
Did you try to use the exe from my Releases? If yes, what happened when you ran it?

**Comment by Dimsok** [comment]
Yes, i did. Even the oldest one. Just said it's not win32 application.

**Comment by FRex** [comment]
I see. If you have access to Python 2.7 on that Windows XP, I can provide a script that does the same as my program. But as you know, XP is way out of support date.

**Comment by Dimsok** [comment]
I have python 2.7, so if you will share the script, i will be glad to try it. All python scripts i have met for zso was only for 3.0.

**Comment by FRex** [comment]
Here it is: https://github.com/FRex/csoinfo/blob/master/csoinfo.py
It worked on 2.7 that I have, works the same as the C program.

**Comment by Dimsok** [comment]
Thanks, i will try.
